mqtt/mqtt-bridge/Cargo.toml

```diff
@@ -7,24 +7,28 @@ edition = "2018"
 [dependencies]
 async-trait = "0.1"
 anyhow = "1.0"
+bincode = "1.2"
 bytes = "0.5"
 config = { version = "0.10", features = ["json"], default-features = false }
 futures-util = "0.3"
 humantime = "2.0"
 humantime-serde = "1.0"
 parking_lot = "0.11.0"
+rocksdb = "0.15.0"
 mqtt3 = { path = "../mqtt3", features = ["serde1"] }
 serde = { version = "1.0", features = ["derive", "rc"] }
 serde_derive = "1.0"
 serial_test = "0.4"
 thiserror = "1.0"
 tracing = "0.1"
 tokio = { version = "0.2", features = ["sync"] }
+uuid = { version = "0.8", features = ["v4"] }
```

Comment:
why?

Reply:
generating new column families for rocksdb databases